backend/src/common/enums/app/route-path.enum.ts

```diff
@@ -2,6 +2,7 @@ enum RoutPath {
   MAIL_ACTIVATE = '/user/verification/:link',
   SEND_ACTIVATE_LINK = '/user/verification',
   NEW_CARS = '/new/cars',
+  WHERE_BUY = '/where/buy',
```

Comment:
We won't have a separate page for this component. Did you add this one only for the testing?

Reply:
yes, I added this page just for testing.